src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-tablecell.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import AlloyEditor from 'alloyeditor';
+
+export default class EzBtnTableCell extends AlloyEditor.ButtonTableCell {
+    static get key() {
+        return 'eztablecell';
+    }
+
+    /**
+     * Lifecycle. Renders the UI of the button.
+     *
+     * @method render
+     * @return {Object} The content which should be rendered.
+     */
+    render() {
+        let buttonCommandsList;
+        let buttonCommandsListId;
+
+        if (this.props.expanded) {
```

Comment:
I would move it to a separate method to make `render` function body as small as possible.

Reply:
Most of the `render` function is copied from AlloyEditor, I would like to keep it like they do for easier maintenance.

Reply:
Ok, let's keep it then.